tests/test_utils.py

```diff
@@ -46,3 +48,39 @@ def test_get_env_key_priority(self):
         os.environ['DATADOG_REQUESTS_DISTRIBUTED_TRACING'] = 'lowest'
         value = get_env('requests', 'distributed_tracing')
         eq_(value, 'highest')
+
+    def test_deprecation_formatter(self):
+        # ensure the formatter returns the proper message
+        msg = format_message(
+            'deprecated_function',
+            'use something else instead',
+            '1.0.0',
+        )
+        expected = "'deprecated_function' is deprecated and will be remove in future versions (1.0.0). use something else instead"
+        eq_(msg, expected)
+
+    def test_deprecation(self):
+        # ensure `deprecation` properly raise a DeprecationWarning
+        with warnings.catch_warnings(record=True) as w:
```

Comment:
tested as suggested in the stdlib: https://docs.python.org/3/library/warnings.html#testing-warnings